packages/core/useDraggable/index.ts

```diff
@@ -135,9 +141,10 @@ export function useDraggable(target: MaybeComputedRef<HTMLElement | SVGElement |
   }
 
   if (isClient) {
-    useEventListener(draggingHandle, 'pointerdown', start, true)
-    useEventListener(draggingElement, 'pointermove', move, true)
-    useEventListener(draggingElement, 'pointerup', end, true)
+    const useCapture = resolveUnref(options?.useCapture ?? true)
+    useEventListener(draggingHandle, 'pointerdown', start, useCapture)
+    useEventListener(draggingElement, 'pointermove', move, useCapture)
+    useEventListener(draggingElement, 'pointerup', end, useCapture)
```

Comment:
```suggestion
    let cleanups: Fn[] = []
    watch(resolveRef(options.useCapture), (useCapture = true) => {
      cleanups.forEach(cleanup => cleanup())
      cleanups = [
        useEventListener(draggingHandle, 'pointerdown', start, useCapture),
        useEventListener(draggingElement, 'pointermove', move, useCapture),
        useEventListener(draggingElement, 'pointerup', end, useCapture),
      ]
    }, { immediate: true })
```
eventListeners should be re-registered when `useCapture` ref value changes.

Reply:
Thank you for your suggestion!